Apps/AdminWebClient/src/ClientApp/src/views/Feedback.vue

```diff
@@ -0,0 +1,152 @@
+              </template>
+              <template v-slot:item.isReviewed="{ item }">
+                <td>
+                  <v-btn class="mx-2" dark small @click="markReviewed(item)">
+                    <v-icon color="green" dark v-if="item.isReviewed">fa-check</v-icon>
+                    <v-icon color="red" dark v-if="!item.isReviewed">fa-times</v-icon>
+                  </v-btn>
+                </td>
+              </template>
+            </v-data-table>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import Vuetify, { VLayout } from "vuetify/lib";
+import { IUserFeedbackService } from "@/services/interfaces";
+import UserFeedback from "@/models/userFeedback";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.config";
+import LoadingComponent from "@/components/Loading.vue";
+import BannerFeedbackComponent from "@/components/BannerFeedback.vue";
+import BannerFeedback from "@/models/bannerFeedback";
+import { ResultType } from "@/constants/resulttype";
+
+@Component({
+  components: {
+    LoadingComponent,
+    BannerFeedbackComponent
+  }
+})
+export default class FeedbackView extends Vue {
+  private isLoading: boolean = true;
+  private showFeedback: boolean = false;
+  private bannerFeedback: BannerFeedback = {
+    type: ResultType.NONE,
+    title: "",
+    message: ""
+  };
+
+  private tableHeaders: any[] = [
+    {
+      text: "Date",
+      value: "createdDateTime",
+      width: '20%'      
+    },
+    {
+      text: "Satisfied?",
+      value: "isSatisfied",
+      width: '10%'      
+    },
+    {
+      text: "Comments",
+      value: "comments",
+      width: '65%'      
+    },
+    {
+      text: "Reviewed?",
+      value: "isReviewed",
+      width: '5%'
+    }
+  ];
+
+  private feedbackList: UserFeedback[] = [];
+
+  private userFeedbackService!: IUserFeedbackService;
+
+  mounted() {
+    this.userFeedbackService = container.get(
+      SERVICE_IDENTIFIER.UserFeedbackService
+    );
+
+    this.loadFeedbackList();
+  }
+
+  private loadFeedbackList() {
+    this.userFeedbackService
+      .getFeedbackList()
+      .then(userFeedbacks => {
+        this.feedbackList = [];
+        this.feedbackList.push(...userFeedbacks);
+      })
+      .catch(err => {
+        this.showFeedback = true;
+        this.bannerFeedback = {
+          type: ResultType.Error,
+          title: "Error",
+          message: "Error loading user feedbacks"
+        };
+      })
+      .finally(() => {
+        this.isLoading = false;
+      });
+  }
+
+  private formatDate(date: Date): string {
+    return new Date(Date.parse(date + "Z")).toLocaleString();
+  }
+
+  private markReviewed(feedback: UserFeedback): void {
+    this.isLoading = true;
+    this.userFeedbackService
+      .markReviewed(feedback.id, feedback.version)
```

Comment:
since it could be un-reviewing "toggleReview" would make more sense. Also on the parameters we will need if its being reviewed or not